x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts

```diff
@@ -78,7 +76,18 @@ export async function getServiceTransactionGroups({
           bool: {
             filter: [
               { term: { [SERVICE_NAME]: serviceName } },
-              { term: { [TRANSACTION_TYPE]: transactionType } },
+              {
+                bool: {
+                  should: [
```

Comment:
filter
```suggestion
                  filter: [
```

Reply:
From what I understand the `filter` acts like and `AND` and in this particular case I'm looking for an `OR`

Reply:
You are absolutely right. I misunderstand what was going on. So if I understand correctly: Previously the `_other` bucket would be filtered out due to the missing `transaction.type` field. This fixes it. 

Can you add a code comment to clarify the behaviour? And add an API test (or verify we already have coverage) to make sure we filter correctly by `transaction.type` for "normal" transaction groups  and that the other group is retrieved even though it doesn't contain `transaction.type`.

Reply:
I'll address this in a following PR